### ISSUE TITLE
Batch Normalization after Convolution

### ISSUE THREAD

**Issue by Kautenja** [Kautenja/keras-pyramid-pooling-module#3]
The original implementation of the module features a batch normalization unit following each convolutional unit. To merge this functionality with this layer, the batch normalization logic will need to be adapted from the default `BatchNormalization` class [here](https://github.com/keras-team/keras/blob/master/keras/layers/normalization.py).

**Comment by KaterinaBao** [comment]
The SPP can be used for various sizes of images. How to achieve that?
